scenario/elasticreload.go

```diff
@@ -177,6 +188,11 @@ forLoop:
 					actionState.AddErrors(errors.Errorf("reload result event contains reload id of unexpected type<%T> value<%v>", eventReloadIDEntry, eventReloadIDEntry))
 					break forLoop
 				}
+
+				_ = sessionState.Rest.GetAsync(
+					fmt.Sprintf("%s/%s?appId=%s", host, getReloadEndpoint, event.ResourceID),
+					actionState, sessionState.LogEntry, nil)
```

Comment:
Isn't this the part they really really didn't want us to do? it's the result on the reload right? which you only would see if your on the reload details page? The argument was that most people won't be.

Reply:
same with the one added to event.

Reply:
>Reloading from hub view and reload history view have this request flow. Have not looked closer at scheduled reloads.

is even this one in hub now even if user stays in hub until reload is done?

Reply:
It is a list of reload results. It is done also when reloading from the meatball menu in the hub.

Reply:
> 
> 
> It is a list of reload results. It is done also when reloading from the meatball menu in the hub.

and where are not talking about the meatball menu in "details" right?